src/setup-beam.js

```diff
@@ -449,6 +478,19 @@ async function get(url0, pageIdxs) {
   return ret
 }
 
+async function getWithMirrors(resourcePath, hexMirrors) {
+  if (hexMirrors.length === 0) {
+    throw new Error(`Could not fetch ${resourcePath} from any hex mirror`)
```

Comment:
```suggestion
    throw new Error(`Could not fetch ${resourcePath} from any hex.pm mirror`)
```

Reply:
Please don't resolve comments, to ease the review process, as previously requested.

Reply:
GitHub automatically resolves comments when the suggestion is committed.